OutOfSchool/Directory.Packages.props

```diff
@@ -1,127 +1,128 @@
-        <PackageVersion Include="System.Linq.Async" Version="6.0.1"/>
-        <!-- Application Performance Monitoring -->
-        <PackageVersion Include="Elastic.Apm.AspNetCore" Version="$(ElasticApmVersion)" />
-        <PackageVersion Include="Elastic.Apm.EntityFrameworkCore" Version="$(ElasticApmVersion)" />
-        <PackageVersion Include="Elastic.Apm.Elasticsearch" Version="$(ElasticApmVersion)" />
-        <PackageVersion Include="Elastic.Apm.StackExchange.Redis" Version="$(ElasticApmVersion)" />
-    </ItemGroup>
+  <PropertyGroup>
+    <BaseVersion>8.0.0</BaseVersion>
+    <EFCoreVersion>8.0.1</EFCoreVersion>
+    <OpenIdDictVersion>4.7.0</OpenIdDictVersion>
+    <SwaggerVersion>6.3.1</SwaggerVersion>
+    <QuartzVersion>3.5.0</QuartzVersion>
+    <RedisVersion>6.0.21</RedisVersion>
+    <AutoMapVersion>12.0.1</AutoMapVersion>
+    <ElasticApmVersion>1.27.1</ElasticApmVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <!--Basic Microsoft Packages-->
+    <PackageVersion Include="Microsoft.AspNet.Identity.Core" Version="2.2.4" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration.Abstractions" Version="$(BaseVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="$(BaseVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="$(BaseVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="$(BaseVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.Options" Version="8.0.1" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration.UserSecrets" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Hosting" Version="6.0.1" />
+    <!--MVC-->
+    <!-- TODO: Isn't AspNetCore.Mvc.Core included by default for a Web project?-->
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="8.0.1" />
+    <PackageVersion Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.AspNetCore.HeaderPropagation" Version="8.0.1" />
+    <!--Entity Framework-->
+    <PackageVersion Include="Microsoft.EntityFrameworkCore" Version="$(EFCoreVersion)" />
+    <PackageVersion Include="Microsoft.EntityFrameworkCore.Design" Version="$(EFCoreVersion)" />
+    <PackageVersion Include="Microsoft.EntityFrameworkCore.Proxies" Version="$(EFCoreVersion)" />
+    <PackageVersion Include="Microsoft.AspNetCore.DataProtection.EntityFrameworkCore" Version="$(EFCoreVersion)" />
+    <PackageVersion Include="Microsoft.EntityFrameworkCore.Tools" Version="$(EFCoreVersion)" />
+    <!-- TODO: Figure out why we need SQLITE :)-->
+    <PackageVersion Include="Microsoft.EntityFrameworkCore.Sqlite" Version="$(EFCoreVersion)" />
+    <PackageVersion Include="MockQueryable.EntityFrameworkCore" Version="7.0.1" />
```

Comment:
Version should be specified using `$(EFCoreVersion)` variable - see line before

Reply:
The latest version of this library is 7.0.1, so we cannot change it to $(EFCoreVersion)

Reply:
Yes, I was wrong, this library is not part of EF versioning.